src/SourceBuild/tarball/content/repos/installer.proj

```diff
@@ -10,6 +10,10 @@
     <OverrideTargetRid Condition="'$(TargetOS)' == 'OSX'">osx-x64</OverrideTargetRid>
     <OSNameOverride>$(OverrideTargetRid.Substring(0, $(OverrideTargetRid.IndexOf("-"))))</OSNameOverride>
 
+    <!-- Determine target portable rid based on bootstrap SDK's portable rid -->
+    <_platformIndex>$(NETCoreSdkPortableRuntimeIdentifier.LastIndexOf('-'))</_platformIndex>
+    <PortableOS>$(NETCoreSdkPortableRuntimeIdentifier.Substring(0, $(_platformIndex)))</PortableOS>
```

Comment:
It'd be nice if  `<PortableOS>` could be set only if it is empty. Thus:
`<PortableOS Condition="'$(PortableOS)' == ''">` 

This ensures that we can override the PortableOS if, for example, it was previously the wrong one.

Reply:
> This ensures that we can override the PortableOS if, for example, it was previously the wrong one.

Do you know can override it if by setting `/p:PortableOS`, for example in `SourceBuild.props`, even when the `Condition` is not present?

Reply:
In depends where that override flows down. I suspect that in `SourceBuild.props`, any non-conditionnal setting of `PortableOS` would override what's in `installer.proj`. For Alpine, our version `6.0.110` of the SDK regressed in that  `--use-current-runtime` came out as `alpine-3.17-x64`. We had thus to override whatever would come out of `$(NETCoreSdkPortableRuntimeIdentifier)`, as it wouldn't be `linux-musl-x64`, for building `6.0.111` by setting `/p:PortableOS=linux-musl` as an argument to `build.sh`.

Reply:
@ayakael is it good for you as is, or do you still want me to add the `Condition`?

Reply:
It'd be more prudent, but it's fine as-is. Once the new Portable rid flows down the condition isn't necessary anyways.